sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/CaseApiService.java

```diff
@@ -37,7 +37,7 @@ public CaseApiService() {}
 
   public Case buildGeneratedCase(Person person) {
     return Case.builder()
-        .disease(DiseasesValues.getRandomDiseaseName())
+        .disease(DiseasesValues.CORONAVIRUS.getDiseaseName())
```

Comment:
I just reverted the changes that Michal did, because all the test before his change to random disease were based on case created through api with disease hardcoded to „Covid-19”. Many Of my tests were failing because Of his change.